### DISCUSSION TITLE
customizing dango rain addon makes it disappear

### DISCUSSION THREAD

**Discussion by CaptainJokes** [ScratchAddons/ScratchAddons#5457] [Help] [answered]
### Description

hello! i'm making a pfp trend of myself, and i made a fork of Scratch Addons and customized dango rain to replace the dango, and when setting it up and installing the exstention, i didn't even appear, even when searching for words like jokes, dango, rain, dan!

### Steps to reproduce

1. Fork*
2. Customize dango-rain (svg and dango.svg)*
3.load it in a web browser you truly love*
4. search for it*
5. JOIN THE CAPTAINJOKES PFP TREND (optional)

### Environment

Browser: Google Chrome
Scratch Addons version: My NO DANGO FOR U fork
Operating system: MacOS Catalina 10.15.7

**Comment by Samq64** [comment]
Should this be moved to discussions?

**Comment by Samq64** [comment]
It works for me when going to https://scratch.mit.edu/users/world_languages/ (who has "dango" in his what I'm working on) but the new image is cropped, so you might want to resize it to the same size as the original or edit the code. You'll also need to edit the code if you want to check for something other than the word "dango".

**Comment by CaptainJokes** [comment]
i tried to search it, but i can't find it? can you send me a gif here so i can see how to search for it.

**Comment by CaptainJokes** [comment, marked as answer]
I fixed it already. now to make changes!
